elements/rh-footer/RhFooter.ts

```diff
@@ -42,38 +42,13 @@ function isHeader(tagName: string) {
  * @csspart logo
  * @slot    social-links
  * @csspart social-links
- * @slot    social-links-start
- * @csspart social-links-start
- * @slot    social-links-end
- * @csspart social-links-end
```

Comment:
I'm removing these regions as social links will no longer be default in the component.